### ISSUE TITLE
Documentation wrong for FileService

### ISSUE THREAD

**Issue by paulmarshall** [Azure/azure-storage-node#413]
At the page https://www.npmjs.com/package/azure-storage#file-storage, the code says:
```
fileService.createDirectoryIfNotExists('taskshare', 'taskdirectory', function(error, result, response) {
  if (!error) {
    // if result = true, share was created.
    // if result = false, share already existed.
  }
});
```

The comment should say:
```
if result.created = true, share was created.
if result.created = false, share already existed.
```

This applies to various examples

**Comment by XiaoningLiu** [comment]
@paulmarshall Thanks! You're right, will fix it.

**Comment by XiaoningLiu** [comment]
Fixed in 2.8.2, thanks!
